packages/ckeditor5-dev-env/lib/release-tools/utils/getnewreleasetype.js

```diff
@@ -5,7 +5,8 @@
 
 'use strict';
 
-const { commitTypes, transform } = require( '../changelog/writer-options' );
+const { availableCommitTypes } = require( './transform-commit-utils' );
+const transformCommitForCKEditor5Package = require( './transformcommitforckeditor5package' );
```

Comment:
Since a specific "CKEditor 5 package" related "transform commit" function is used here... isn't this function related to "CKEditor 5 package" too?

Wouldn't it make sense if `transformCommit` was this function's param?